osf/migrations/__init__.py

```diff
@@ -123,6 +126,13 @@ def update_permission_groups(sender, verbosity=0, **kwargs):
         update_provider_auth_groups(verbosity)
 
 
+def update_waffle_flags(sender, verbosity=0, **kwargs):
+    if getattr(sender, 'label', None) == 'osf':
+        if 'pytest' not in sys.modules:
```

Comment:
@Johnetordoff what's the check for local environment only? I think the fix is to avoid `server` in addition to CI?